src/main/java/frc/robot/subsystems/Hopper.java

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.LoadingConstants;
+
+public class Hopper extends SubsystemBase {
+
+  private final TalonSRX m_hopperMotor;
+
+  /**
+   * Creates a new Hopper.
+   */
+  public Hopper() {
+    m_hopperMotor = new TalonSRX(LoadingConstants.kLoadingMotorPort);
+  
+    motorInit(m_hopperMotor);
+    
+  }
+
+  public void motorInit(TalonSRX motor) {
```

Comment:
While it's not something that I've explicitly told people to do or not do, I used it in the sample project that I suggested they work off of, and it serves as a solid template.  I agree that it shouldn't be necessary here, but it's a good habit to get into, since we will definitely want to add more things here, like setting current limits, inversion, etc.